examples/gallery/symbols/patterns.py

```diff
@@ -1,67 +1,44 @@
-r"""
+"""
 Bit and hachure patterns
 ========================
 
-PyGMT allows using bit or hachure patterns via the ``fill`` parameter
-or similar parameters:
+In addition to solid colors, PyGMT also allows using bit and hachure patterns to fill
```

Comment:
I am personally prefer removing "solid", as it sounds a little bit strange to me (there are no fluid colors...).

```suggestion
In addition to colors, PyGMT also allows using bit and hachure patterns to fill
```